files/sidebars/performancesidebar.yaml

```
@@ -50,3 +50,9 @@ l10n:
     Reference: リファレンス
     PerformanceAPIs: パフォーマンス API
     RelatedAPIs: 関連 API
+  zh-CN:
+    Tutorials: 教程
+    Learn_web_performance: 学习 Web 性能
+    Reference: 参考
+    PerformanceAPIs: Performance API
+    RelatedAPIs: 相关 API
```

We can translate the title "Performance API" and distinguish the format of the context to avoid confusion.

```suggestion
    PerformanceAPIs: 性能 API
    RelatedAPIs: 相关的 API
```